src/main/java/spoon/reflect/factory/Factory.java

```diff
@@ -130,6 +131,67 @@
  * Otherwise FactoryImpl is a default implementation.
  */
 public interface Factory {
+	String[] keywords = {
```

Comment:
Why do you add that internal code into Factory interface? I suggest to move it to an utility class in a ...internal... package.
WDYT?

Reply:
Yes no problem about moving it: I was wondering about putting it in `support.util` but then I'd have to create a new class for it. Unless you have a better idea?

Reply:
I am personally OK with new utility class... but may be there is someone who has better idea... utility classes are not the best practice.